Description: <fix Unable to automatically mount non-system partitions>

 udisks2 (2.9.4-deepin) stable; urgency=medium
 .
   * ['Rebuild  against by  current gcc 11.2']
Author: Deepin Packages Builder <packages@deepin.org>

---

Origin: <vendor>
Bug: <https://github.com/orgs/linuxdeepin/discussions/3974>
Bug-deepin: https://github.com/linuxdeepin/developer-center/issues
Last-Update: 2023-04-07

--- udisks2-2.9.4.orig/data/org.freedesktop.UDisks2.policy.in
+++ udisks2-2.9.4/data/org.freedesktop.UDisks2.policy.in
@@ -29,7 +29,7 @@
     <defaults>
       <allow_any>auth_admin</allow_any>
       <allow_inactive>auth_admin</allow_inactive>
-      <allow_active>auth_admin_keep</allow_active>
+      <allow_active>yes</allow_active>
     </defaults>
   </action>
 
